controllers/datadogagent/feature/sbom/feature_test.go

```diff
@@ -106,6 +106,10 @@ func Test_sbomFeature_Configure(t *testing.T) {
 				Name:  apicommon.DDSBOMHostEnabled,
 				Value: "true",
 			},
+			{
+				Name:  apicommon.DDHostRootEnvVar,
+				Value: "/host",
+			},
```

Comment:
could you add in the test the validation that the volumes are mounted in host sbom is enabled?